README.md

```diff
@@ -1,20 +1,22 @@
 # sublime-parquet
-Reads Apache Parquet files in Sublime Text. The records are shown in JSON format, one JSON object per line. Parquet files are opened in read-only mode.
+Reads Apache Parquet files in Sublime Text. Can use several mechanisms for decoding - seee requirements.
 
 # Screenshot
-![screenshot](https://raw.github.com/yuj/sublime-parquet/master/screenshot.png)
+![screencap](https://raw.github.com/dogversioning/sublime-parquet-python/main/screencap.png)
 
 # Installation
 **via Package Control**
 
 1. Make sure you have [Package Control](https://packagecontrol.io/installation) installed.
-1. Open the Command Palette (`command-shift-P` on macOS; `ctrl-shift-P` on Ubuntu) and choose _Install Package_.
-1. Choose _Parquet_ from the list.
+1. Open the Command Palette (`command-shift-P` on macOS; `ctrl-shift-P` on Ubuntu/Windows) and choose _Install Package_.
+1. Choose _ParquetPython_ from the list.
```

Comment:
Probably still need to stay with _Parquet_ so the the Sublime Text Package Control can still find this plugin.